psydac/fem/vector.py

```diff
@@ -606,15 +634,23 @@ def eval_fields_irregular_tensor_grid(self, grid, *fields, weights=None, overlap
 
     # ...
     def eval_field_gradient( self, field, *eta ):
-        raise NotImplementedError( "ProductFemSpace not yet operational" )
+        raise NotImplementedError( "MultipatchFemSpace not yet operational" )
 
     # ...
     def integral( self, f ):
-        raise NotImplementedError( "ProductFemSpace not yet operational" )
+        raise NotImplementedError( "MultipatchFemSpace not yet operational" )
 
     #--------------------------------------------------------------------------
     # Other properties and methods
     #--------------------------------------------------------------------------
+    @property
+    def is_multipatch(self):
+        return True
+
+    @property
+    def is_vector_valued(self):
+        return self.patch_spaces[0].is_vector_valued
```

Comment:
These two properties should be moved up, to the section "Abstract interface: read-only attributes".

Reply:
Done in 55fd2c9